### ISSUE TITLE
Extension List : Accessibility: Allow Tab to Current List-Line Buttons and Action Items

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#32033]
Currently in the extension sidebar list keyboard navigation of the list is separate from the tab index
which controls access to the buttons and action items. A keyboard user cannot navigate
to a line and its action bar items you have to tab from the top.
See:
#24119 

Proposed Solution:

- adding an action bar option for external tabindex control
- allow the the upper level container controller to use focus and selection
to control the tabindexes
- the extension action icons are now only enabled when focused or selected
- before this one had to go through the entire tabindex sequence
to go from a selected extension to using the action icons

I would like to update and resubmit my original PR if the above sounds good.
https://github.com/Microsoft/vscode/pull/24628

**Comment by cleidigh** [comment]
I think a variety of changes have essentially solved this.

Closing.
